### ISSUE TITLE
puts in option.rb on rubygems.org

### ISSUE THREAD

**Issue by rforte** [trailblazer/trailblazer-context#9]
0.3.2 version on rubygems contains a 'puts' that doesn't appear in master.

```
    # Generic builder for a callable "option".
    # @param call_implementation [Class, Module] implements the process of calling the proc
    #   while passing arguments/options to it in a specific style (e.g. kw args, step interface).
    # @return [Proc] when called, this proc will evaluate its option (at run-time).
    def self.build(call_implementation, proc)
      if proc.is_a? Symbol
        ->(*args, &block) { call_implementation.evaluate_method(proc, *args, &block) }
      else
        ->(*args, &block) {

puts "@@@@@ #{proc.inspect}"
         call_implementation.evaluate_callable(proc, *args, &block) }
      end
    end
```

Not a huge deal other than it makes the logs a bit verbose and rubygems is out of sync with what's in the repo here.

**Comment by apotonick** [comment]
Oh, my bad! Thanks @rforte! :beers:

**Comment by apotonick** [comment]
Closed with 0.3.3 release.

**Comment by rforte** [comment]
Danke duder!
